docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<p>(1) Is there extra space between the first two sentences?</p><p>(2)<code>According to...,</code> note that I replace the dot with comma</p><p>(3) please modify the second sentence as follows:</p><pre class="ql-syntax" spellcheck="false">.... of premature venticular contractions, with start index of 580 and 2030 (see figure below).
</pre><p> </p><p>(4) please modify the third sentence as follows:</p><pre class="ql-syntax" spellcheck="false">Those are the two instances we would like to discover.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>